content/en/tracing/profiler/search_profiles.md

```diff
@@ -184,6 +197,22 @@ Once enabled, the following profile type is collected:
 Wall Time
 : Shows the elapsed time spent in managed methods. Elapsed time includes time when code is running on CPU, waiting for I/O, and anything else that happens while the method is running. 
 
+{{< /programming-lang >}}
+{{< programming-lang lang="php" >}}
+
+Once enabled, the following profile types are collected:
+
+Wall Time
+: Shows the elapsed time used by each function. Elapsed time includes time when code is running on CPU, waiting for I/O, and anything else that happens while the function is running.
+
+{{< /programming-lang >}}
+{{< programming-lang lang="linux" >}}
+
+Once enabled, the following profile types are collected:
+
+CPU
+: Shows the time each function spent running on the CPU, including Ruby and native code.
```

Comment:
This is a bit confusing, since the native profiler will surface the internals of the Ruby runtime, but that's slightly different from what a user would think of as "Ruby code."  Is it clearer if we don't qualify the statement by just saying "Shows the time each function spent running on the CPU?"

Reply:
Nice catch, bad copy/pasta.